src/test/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/UpdateClauseDeserializerTest.java

```diff
@@ -123,5 +123,26 @@ public void mustHandleSubdoc() throws Exception {
       UpdateClause updateClause = objectMapper.readValue(json, UpdateClause.class);
       assertThat(updateClause.buildOperations()).hasSize(1).contains(operation);
     }
+
+    @Test
+    public void mustHandleDate() throws Exception {
```

Comment:
Can you add a  test case to show invalid date format is handled.
`{"$set" : {"dateType": {"$date": "2023-01-01T00:00:00Z"}}}`

Reply:
I can do that. It needs to go in integration test I think, but I'll see.

Reply:
Added an IT (validation occurs in Shredder, not yet during deserialization)